maths/extended_gcd.nim

```diff
@@ -0,0 +1,76 @@
+## Extended Euclidean algorithm
+##
+## Provides the implementation of the function extendedGCD*
+
+import std/math
+
+
+func updateCoefficients(t0, t1, q: int): (int, int) =
+  (t1, t0 - q * t1)
```

Comment:
Is this function really necessary? I like putting in a function something that appears twice at the only condition that the name of the function is explicit. This is not the case here. What kind of update does it perform?

Reply:
Yes, it is - it shows (as clearly as possible) that the `x`'s and `y`'s evolve in the same way.

I am also not entirely happy with the name. Do you have some suggestions?